core/primitives/src/block.rs

```diff
@@ -387,6 +394,10 @@ pub struct Block {
     pub header: BlockHeader,
     pub chunks: Vec<ShardChunkHeader>,
     pub challenges: Challenges,
+
+    // Data to confirm the correctness of randomness beacon output
+    pub vrf_value: near_crypto::vrf::Value,
+    pub vrf_proof: near_crypto::vrf::Proof,
```

Comment:
Why there are two fields `vrf_value` and `random_value` in different places?

Reply:
The logic is the following: with this randomness beacon the `random_value = hash(vrf_value)`.
However, once we switch to the new randomness beacon, that will change to `random_value = new_beacon(hash(vrf_value))`. In other words, `vrf_value` will always stay the output of the VRF by the current block producer, while the way the `random_value` is computed can change.